pkg/types/args_test.go

```diff
@@ -0,0 +1,113 @@
+package types_test
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/appc/cni/pkg/types"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestArgs(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "pkg/ns Suite")
+}
+
+var _ = Describe("UnmarshallableBool UnmarshalText", func() {
+	positive := []string{"True", "true", "1"}
+	Context(fmt.Sprintf("When passed one of %q", positive), func() {
+		It("should result in boolean true", func() {
+			ubexp := types.UnmarshallableBool(true)
+			for _, s := range positive {
```

Comment:
It looks like you want these to be table based tests (which is awesome). Ginkgo has a built in to do this.  You can check it out [here](https://onsi.github.io/ginkgo/#extensions)